versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_version = "~> 0.12.0"
 
   required_providers {
-    aws      = "~> 2.0"
+    aws      = ">= 2.34.0"
```

```suggestion
    aws      = "~> 2.3"
```

we don't want it to automatically start using version `3.x.x` which could have breaking changes

---

fair enough. I'll fix it.

---

I set the minimum allowed version for the AWS provider `~> 2.34` since the `image_scanning_configuration` feature was released in that version is not going to work with prior versions of the AWS provider.